perf/collector_libpfm.go

```diff
@@ -214,6 +206,34 @@ func createPerfEventAttr(event CustomEvent) *unix.PerfEventAttr {
 	return config
 }
 
+func getPerfEventAttr(name string) (*unix.PerfEventAttr, error) {
```

Comment:
It seems to me that this is the only meaningful change in this file now. Everything else can be undone (see my comment to `collector.cpuFiles`).

Reply:
See comment above.